scripts/run_pylint.py

```diff
@@ -21,6 +21,8 @@
 violations (hence it has a reduced number of style checks).
 """
 
+from __future__ import print_function
```

Comment:
Missed use of `print` statement at line 152, which broke the `lint` test on Travis.